src/expandableCalendar/index.tsx

```diff
@@ -530,7 +530,7 @@ const ExpandableCalendar = (props: ExpandableCalendarProps) => {
           firstDay={firstDay}
           {...others}
           allowShadow={disableWeekScroll ? undefined : false}
-          current={date}
+          current={disableWeekScroll ? date : undefined}
```

Comment:
When adding `disableWeekScroll` prop to the ExpandableCalendar the arrows stop working.

Reply:
it already exists in the code, all i did was to pass current only for single week view instead of WeekCalendar (it is required for week but in WeekCalednar we get the date from the context which means that this prop updates for no reason

Reply:
Currently, `disableWeekScroll` prop makes the calendar not really functional. This is a regression we should address here.